unskript-ctl/bash_completion_unskript_ctl.bash

```diff
@@ -6,6 +6,7 @@ _unskript-client-completion() {
     COMPREPLY=()
     cur="${COMP_WORDS[COMP_CWORD]}"
     prev="${COMP_WORDS[COMP_CWORD-1]}"
+    connector_list=("aws" "k8s" "postgres" "mongodb" "elasticsearch" "vault" "ssh" "keycloak" "github" "redis")
```

Comment:
to keep this list consistent with the add_creds.py, we can put the connector list in a file and make add_creds and this bash read from there

Reply:
I will raise a ticket to track the issue.